sdk-commons/src/main/java/org/openmrs/maven/plugins/model/BaseSdkProperties.java

```diff
@@ -134,9 +135,23 @@ public List<Artifact> getWarArtifacts(){
         return  artifactList;
     }
 
-    protected Set<Object> getAllKeys(){
-        return properties.keySet();
-    }
+	public List<Artifact> getConfigArtifacts() {
+		List<Artifact> artifactList = new ArrayList<>();
+		for (Object keyObject : getAllKeys()) {
+			String key = keyObject.toString();
+			String artifactType = getArtifactType(key);
+			if (artifactType.equals(TYPE_CONFIG)) {
+				artifactList.add(
+						new Artifact(checkIfOverwritten(key, ARTIFACT_ID), getParam(key), checkIfOverwritten(key, GROUP_ID),
+								checkIfOverwritten(key, TYPE)));
+			}
+		}
+		return artifactList;
+	}
+
+	protected Set<Object> getAllKeys() {
```

Comment:
What is the use of this method?

Reply:
It is not a method I wrote. It returns all the keys from the properties.

Reply:
Oh ok.